ui/StatusQ/src/StatusQ/Components/StatusChatListItem.qml

```diff
@@ -187,19 +187,22 @@ Rectangle {
                 visible: mutedIconSensor.containsMouse
             }
         }
-
-        StatusBadge {
-            id: statusBadge
-
-            anchors.verticalCenter: parent.verticalCenter
+        Item {
+            width: 32
+            height: parent.height
             anchors.right: parent.right
             anchors.rightMargin: 8
-
-            color: root.muted ? Theme.palette.primaryColor2 : Theme.palette.primaryColor1
-            border.width: 4
-            border.color: color
-            value: root.notificationsCount
-            visible: root.notificationsCount > 0
+            StatusBadge {
```

Comment:
Why a nested StatusBadge inside another Item?

Reply:
As said above, when we only have unread messages we need a rounded rect of 8x8, but if other channels have mentions this rect would become 22x22, 30x30 etc. as we need these to be aligned horizontally I decided wrap into an Item with fixed sizing